web/templates/search/search.html.eex

```diff
@@ -14,18 +14,21 @@
   <thead>
     <tr>
       <th>rank</th>
-      <th class="text">text</th>
       <th class="caption">transcription</th>
-      <th>click</th>
     </tr>
   </thead>
   <tbody>
 <%= for {tweet, rank} <- @results do %>
     <tr>
       <td><%= round(rank * 10000) %></td>
-      <td><%= tweet.text %></td>
-      <td><%= tweet.caption %></td>
-      <td><%= link "see on Twitter", to: "https://twitter.com/#{tweet.body["user"]["screen_name"]}/status/#{tweet.id_number}" %></td>
+      <td>
+        <p><%= tweet.text %></p>
+        <p><%= tweet.caption %></p>
+        <blockquote class="twitter-tweet" data-lang="en">
+          <%= content_tag :a, href: "https://twitter.com/#{tweet.body["user"]["screen_name"]}/status/#{tweet.id_number}" do %>x<% end %>
+        </blockquote>
+        <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
This script inclusion should be outside of the table, or possibly at the bottom of the app layout template.